app.js

```diff
@@ -901,7 +901,7 @@ io.sockets.on('connection',function(socket){
                 openhab.status = 'online';
                 openhab.last_online = new Date;
                 openhab.openhabVersion = socket.handshake.openhabVersion;
-                openhab.myohVersion = socket.handshake.myohVersion;
+                openhab.clientVersion = socket.handshake.clientVersion;
```

Comment:
@marziman sorry for late late response, lets go ahead and update the model and store the correct field name.  Once this is changed we can go back and remove the field from existing documents.  Also did you see openhabaccesslog.js?  is also has a myopenhab reference you will want to change.